gammapy/background/energy_offset_array.py

```diff
@@ -104,3 +127,76 @@ def plot_image(self, ax=None, offset=None, energy=None, **kwargs):
         image = ax.imshow(self.data, extent=extent, **kwargs)
         plt.colorbar(image)
         return ax
+
+    @classmethod
+    def read(cls, filename):
+        """Create `EnergyOffsetArray` from  fits file.
+
+        Parameters
+        ----------
+        filename : str
+            File name
+        """
+        hdu_list = fits.open(filename)
+        return cls.from_fits(hdu_list)
+
+    
+    @classmethod
+    def from_fits(cls, hdu, scheme=None):
+        """Read EnergyOffsetArray from a fits binary table.
+
+        Parameters
+        ----------
+        hdu : `~astropy.io.fits.BinTableHDU`
+            HDU binary table for the EnergyOffsetArray.
+        Returns
+        -------
+        cube : `~gammapy.background.EnergyOffsetArray`
+            EnergyOffsetArray object.
+        """
+
+        header = hdu[1].header
+        data = hdu[1].data
+        
+        # get offset and energy binning
+        offset_edges = _make_bin_edges_array(data['THETA_LO'], data['THETA_HI'])
+        energy_edges = _make_bin_edges_array(data['ENERG_LO'], data['ENERG_HI'])
+        
```

Comment:
Perhaps this is not good to define Quantity object in the constructor with fixed unit? In this case I can't define Quantity object here for the offset and the energy with the unit read in the fits file?